utils/utils.c

```diff
@@ -428,11 +428,20 @@ gbStrcpy(char *dest, const char *src, size_t destbytes,
     if (n > (destbytes - 1))
       return NULL;
 
-    ret = strncpy(dest, src, n);
-    /* strncpy NULL terminates if the last character is \0.  Therefore
-     * force the last byte to be \0
-     */
+    ret = memcpy(dest, src, n);
     dest[n] = '\0';
 
     return ret;
 }
+
+
+char *
+gbStrcat(char *dest, const char *src, size_t destbytes, size_t srcbytes,
+         const char *filename, const char *funcname, size_t linenr)
+{
+    if (srcbytes > (destbytes - 1))
+      return NULL;
+
+    /* strncat will add the NULL terminates */
+    return strncat(dest, src, srcbytes);
```

Comment:
@lxbsz I still see the use of same old trick here, can we change this to memcpy as well ?

Reply:
@pkalever Updated done.
Thanks.